### ISSUE TITLE
A11y Fail: Input - Add role="alert" for error message.

### ISSUE THREAD

**Issue by olehnoskov-cengage** [cengage/react-magma#1767]
**Describe the bug**
When skipping a required field, an error message appears next to the field. 
This message are not conveyed to assistive technologies when they appear

**Expected behavior**
Add role="alert" to the element containing the error message. 

**Screenshots**

![Image](https://github.com/user-attachments/assets/1bb8d313-cbbc-4471-8f32-60c2fd2e7417)

**Additional context**
[Linked spike](https://github.com/cengage/react-magma/issues/1748)

[Original bug](https://jira.cengage.com/browse/NG-96812)

[Perkins Advice:Using the status role for status messages](https://perkins.caspio.com/dp/ca3640008726de167a974caaba70?Advice_ID=ADVICE_004)

Maybe related to this [ticket](https://github.com/cengage/react-magma/issues/1661)

**Comment by olehnoskov-cengage** [comment]
Yes, you are right!
I thought we should additionally add role="alert" to this field.
I'm going to close this ticket.
